controllers/idler/idler_controller_test.go

```diff
@@ -8,9 +8,15 @@ import (
 	"testing"
 	"time"
 
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
+
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	nstemplatesetTest "github.com/codeready-toolchain/member-operator/controllers/nstemplateset"
```

Comment:
Go package naming conventions discourage the use of underscores and mixedCaps. In this case it doesn't look like `nstemplateset` is colliding with other names in the file so can we just remove the alias?